src/include/btmem.h

```diff
@@ -695,8 +696,9 @@ struct __wt_page {
 #define WT_PAGE_EVICT_LRU_URGENT 0x020u   /* Page is in the urgent queue */
 #define WT_PAGE_EVICT_NO_PROGRESS 0x040u  /* Eviction doesn't count as progress */
 #define WT_PAGE_INTL_OVERFLOW_KEYS 0x080u /* Internal page has overflow keys (historic only) */
-#define WT_PAGE_SPLIT_INSERT 0x100u       /* A leaf page was split for append */
-#define WT_PAGE_UPDATE_IGNORE 0x200u      /* Ignore updates on page discard */
+#define WT_PAGE_PREFETCH 0x100u           /* The page is being pre-fetched */
```

Comment:
Wanted to confirm my understanding, but if this flag has been set does this mean the page has already been read into cache?

Reply:
Yes - this is on the WT_PAGE structure, which only exists when a page has been instantiated (read into cache). The WT_UPDATE structure is the thing that references a page even when it isn't in the cache.